deeprank/generate/DataGenerator.py

```diff
@@ -1614,9 +1540,7 @@ def _get_aligned_sqldb(self, pdbfile, dict_align):
             dict_align['export'] = False
 
         if dict_align['selection'] == 'interface':
-            sqldb = align_interface(pdbfile, plane=dict_align['plane'],
-                                    export=dict_align['export'],
-                                    chain1=self.chain1, chain2=self.chain2)
+            raise ValueError("interface alignment is not supported")
 
         else:
 
```

Comment:
What are aligning input PDB with?

Reply:
The align_interface function comes from pdb2sql, its docstring says:
"align the interface of a complex in a given plane".

Since our version of deeprank won't involve an interface but a mutant, we cannot do this. So I'm disabling this functionality.

Reply:
Actually, we might change the alignment option completely.